### ISSUE TITLE
Slight lag on opening comboboxes

### ISSUE THREAD

**Issue by opsidjflksdf** [ocornut/imgui#7537]
### Version/Branch of Dear ImGui:

Version 1.XX, Branch: XXX (master/docking/etc.)

### Back-ends:

imgui_impl_XXX.cpp + imgui_impl_XXX.cpp

### Compiler, OS:

wasm

### Details:

Hello, 
I'm trying out the DearImgui demo on the web
https://pthom.github.io/imgui_manual_online/manual/imgui_manual.html
and 
https://jnmaloney.github.io/WebGui/imgui.html
I notice there is a slight but noticeable lag between the click on a combobox/drop down arrow and when it opens up. It's not huge, but it's definitely there and enough to faze you off a bit, especially since everything else in the interface feels instantaneous.
I wonder if this is something deliberate, catering to certain usability considerations, or rather a programming artifact, due to implementation details?
I looked at egui and it doesn't have the same lag. 
I would appreciate any pointers on how to possibly eliminate it

Thank you very much for this awesome library!

### Minimal, Complete and Verifiable Example code:

https://pthom.github.io/imgui_manual_online/manual/imgui_manual.html

**Comment by opsidjflksdf** [comment]
I've just built myself a web version, with a newer imgui and this one doesn't have the lag issue. There probably was some work done on combo-boxes and the demos that are out there as bit out of date. I should be all good. Thank you for the hard work that goes into DearImgui!

**Comment by ocornut** [comment]
I don't see any lag in the aforementioned link, and nothing related to this has changed recently.
Perhaps you are referring to the fact that button press validation requires a mouse-down-then-release event, instead of just a mouse-down? That's how a majority of actions works. It might make sense to change the behavior for combo boxes tho.

**Comment by cfillion** [comment]
> I notice there is a slight but noticeable lag between the click on a combobox/drop down arrow and when it opens up. It's not huge,

Seems related to the "Enable idling" checkbox at the bottom right. Combo popups open immediately with that disabled. That extra idling feature is not part of dear imgui, it comes from [this repo](https://github.com/pthom/hello_imgui).

I don't get any delay opening combos in the second link here (Firefox).

**Comment by ocornut** [comment]
Right so it would be a bug or feedback to submit to @pthom, not here.

**Comment by opsidjflksdf** [comment]
I revisited and I have the same experience as @cfillion reported (checking the Idle flag introduces the combo opening lag; unchecking it gets rid of it). 
Thanks! I'll file a ticket with Hello_Imgui.

**Comment by pthom** [comment]
Bonjour,

Since Omar pinged me, I could see this issue. 

The commit [add timeActiveAfterLastEvent ](https://github.com/pthom/hello_imgui/commit/5003a0052e2dbb81cd2426a373eecec53d268e92) should solve the issue since it has a grace period of 3 seconds before idling after the last user event. 

The [manual](https://pthom.github.io/imgui_manual_online/manual/imgui_manual.html) was updated also, and should not exhibit this lag anymore. 

And @ocornut , for info, I just updated the manual to ImGui v1.90.5
